extensions/sql-database-projects/src/test/netCoreTool.test.ts

```diff
@@ -41,8 +42,25 @@ describe('NetCoreTool: Net core install popup tests', function (): void {
 			should(result).true('dotnet is either not present or in /usr/local/share by default');
 		}
 	});
-});
 
-export async function sleep(ms: number): Promise<{}> {
-	return new Promise(resolve => setTimeout(resolve, ms));
-}
+	it('should run a command successfully', async function (): Promise<void> {
+
+		const netcoreTool = new NetCoreTool();
+		const dummyFile = 'dummy.dacpac';
```

Comment:
could use the testUtils to create a temp folder instead of having to manually clean up